sparkbox/stylelint-config-sparkbox
# Make stylelint a peerDependency of stylelint-config-sparkbox

Fixes #22

## Testing Instructions

1. Confirm that the change matches what is expected in [the issue](https://github.com/sparkbox/stylelint-config-sparkbox/issues/22).
2. Verify that the updated docs make sense: https://github.com/sparkbox/stylelint-config-sparkbox/blob/peer-dependency/README.md
3. Pull down this branch.
4. Reinstall all the dependencies (`rm -rf node_modules && npm install`), open the `node_modules` folder, and confirm that `stylelint` is still in there.

**Extra Credit**
1. Create a new folder on your computer, and initialize a project in it using this branch:
```
mkdir test-peerdeps && cd test-peerdeps
npm init -y && npm install sparkbox/stylelint-config-sparkbox#peer-dependency --save-dev
```
2. Confirm that the following message is included as part of the install:
```
@sparkbox/stylelint-config-sparkbox@1.0.3 requires a peer of stylelint@>= 13 but none is installed. You must install peer dependencies yourself.
```
3. Confirm that the `node_modules` folder does NOT contain `stylelint`.
